core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -219,6 +220,28 @@ class UserAuthJsonSupport extends JsonSupport {
   implicit val WorkspaceBillingAccountFormat = jsonFormat2(WorkspaceBillingAccount)
 
   implicit val RawlsBillingProjectResponseFormat = jsonFormat7(RawlsBillingProjectResponse)
+
+  implicit val SpendReportingForDateRangeFormat = jsonFormat5(SpendReportingForDateRange)
+
+  implicit val SpendReportingAggregationKeyFormat = jsonFormat1(SpendReportingAggregationKey)
+  implicit val SpendReportingAggregationFormat = jsonFormat2(SpendReportingAggregation)
+  implicit val SpendReportingResultsFormat = jsonFormat2(SpendReportingResults)
+
+
 }
 
 object UserAuthJsonSupport extends UserAuthJsonSupport
+
+// TODO don't dump these case classes here
```

Comment:
Probably pulling this out to a SpendModel or something like that along with the `BillingProjectSpendExport` from `SpendReportingService`